app/grandchallenge/core/views.py

```diff
@@ -130,6 +155,14 @@ def get_context_data(self, **kwargs):
                     "A platform for end-to-end development of machine "
                     "learning solutions in biomedical imaging."
                 ),
+                "latest_challenges": Challenge.objects.filter(hidden=False)
```

Comment:
Rather than showing the latest challenges and algorithms maybe instead add another model attribute to display them as a highlight, then we have control over what appears there rather than users.

Reply:
That's a nice idea. Who should be able to set this attribute though? If we don't want users to determine this, than it should be a hidden field that can only be set in the admin (i.e., by you), or is there a better option?

Reply:
Yes, add it to the model and `admin.py`, just don't set it in the object create and update views the editors get. We can add the permission bit to edit those to the staff members we want to do this, not just me.